src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -51,6 +53,12 @@ export default class Mutations extends React.Component<IMutationsPageProps, {}>
                         onToggle={this.onToggleOql}
                     />
                 </div>
+                
+                {this.props.store.mutationMapperStores.isComplete && ! this.props.store.mutationMapperStores.result[this.mutationsGeneTab].dataStore.showingAllData &&
+                    <div>
```

Comment:
you probably don't need extra div around this

Reply:
also, try putting this above inside the tabMessageContainer div.

Reply:
Thanks, I just fixed it.